webapp/channels/src/components/invitation_modal/invite_view.scss

```diff
@@ -32,7 +32,7 @@ button.InviteView__copyLink.tertiary-button:focus {
             outline: none;
         }
 
-        &.btn.btn-cancel {
+        &.btn.btn-tertiary {
```

Comment:
This is not applied anymore (actually, it wasn't applied before either) because you remove the `__copyLink` style from existence, so... probably the whole `&__copyLink` can be removed.

Reply:
Done.